modules/LinearBoltzmannSolver/IterativeOperations/lbs_setsource.cc

```diff
@@ -55,16 +55,14 @@ void lbs::SteadySolver::
 
     //==================== Obtain src
     double* src = default_zero_src.data();
-    if (P0_src and apply_fixed_src)
+    if (P0_src && apply_fixed_src)
```

Comment:
I am not sure why the and (old) was changed to && (new). I thought we decided to go with more english-like symbols